ext/console/internal.d.ts

```diff
@@ -4,9 +4,12 @@
 /// <reference lib="esnext" />
 
 declare module "ext:deno_console/01_console.js" {
-  function createFilteredInspectProxy<TObject>(params: {
-    object: TObject;
-    keys: (keyof TObject)[];
-    evaluate: boolean;
```

Comment:
We may need to remain `evaluate` option to detect if it's an instance object or a prototype object.

<img width="405" src="https://github.com/user-attachments/assets/cf9db4ab-07bb-4f88-b9ee-679000db0eb1">